.github/workflows/dependabot-code-gen.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       id: go
     - name: Check out code into the Go module directory
       uses: actions/checkout@v3 # tag=v3.2.0
-    - uses: actions/cache@9b0c1fce7a93df8e3bb8926b0d6e9d89e92f20a7 # tag=v3.0.11
+    - uses: actions/cache@58c146cc91c5b9e778e71775dfe9bf1442ad9a12 # tag=v3.2.3
```

Comment:
I'm not sure why we're not simply using `actions/cache@v3` for this? https://github.com/marketplace/actions/cache#example-cache-workflow

@nawazkh thoughts?